client/scss/style.scss

```diff
@@ -287,7 +287,7 @@ content-row {
 				line-height: 1.4;
 
 				&:last-child {
-					margin-bottom: 0px;
+					// margin-bottom: 0px;
```

Comment:
Haha, sure.  I'm actually still not sure whether or not that change should be there.  I think I made it because something else was broken, so I'm going to revert it.